src/routes/signup-pupil.svelte

```diff
@@ -15,9 +15,8 @@
   import { signup_form_submit_handler } from '../utils/airtable'
 
   export const load: Load = async () => {
-    const chapters = (await fetchChapters()).filter(
-      (chap: Chapter) => chap.acceptsSignups
-    )
+    let chapters = await fetchChapters()
```

Comment:
what does this change? Could it possibly break the signup?

Reply:
Just a refactor for readability. The code is equivalent.